src/Managed.sol

```diff
@@ -183,16 +91,6 @@ abstract contract Managed is UUPSUpgradeable {
         return bytes32(uint256(keccak256(bytes(name))) - 1024);
     }
 
-    /// @notice Returns the Managed storage.
-    /// @return storageRef The storage.
-    function _managedStorage() internal view returns (ManagedStorage storage storageRef) {
-        bytes32 slot = _managedStorageSlot;
-        // slither-disable-next-line assembly
-        assembly {
-            storageRef.slot := slot
-        }
-    }
-
     /// @notice Authorizes the contract upgrade. See `UUPSUpgradeable` docs for more details.
     function _authorizeUpgrade(address /* newImplementation */ ) internal view override onlyAdmin {
```

Comment:
Following the PR description I thought you want to check here if the `proposedAdmin` equals `address(0)`?

Reply:
No, no, the PR description states that the slot changes, so any pending admin address will forever stay in storage without ever being accessed, and from the upgrade forward a different slot will be used to store the pending admin address. I think that it's a good tradeoff for keeping the code clean and the storage layout legacy-free. There hasn't been a single admin upgrade ever performed, so we can probably avoid having a pending admin during the upgrade.

I think that we should accept `address(0)` because it's a nice way to explicitly cancel the admin change proposal, it's obvious and it doesn't require a new function.